components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+  CGSize bannerViewSize =
+      [self.bannerView sizeThatFits:CGSizeMake(CGRectGetWidth(self.view.bounds), CGFLOAT_MAX)];
+
+  // Adjust bannerViewContainer's frame
+  CGFloat topAreaInset = 0.0f;
+  if (@available(iOS 11.0, *)) {
+    topAreaInset = self.view.safeAreaInsets.top;
+  }
+  self.bannerView.frame =
+      CGRectMake(0.0f, topAreaInset, bannerViewSize.width, bannerViewSize.height);
+}
+
 #pragma mark - Internal helpers
 
 - (NSArray<BannerExampleUseInfo *> *)getBannerExampleList {
   NSMutableArray *bannerExampleList = [[NSMutableArray alloc] init];
   BannerExampleUseInfo *exampleUseInfo1 =
       [BannerExampleUseInfo infoWithIdentifier:@"example1"
-                                   displayName:@"Single Line"
+                                   displayName:@"Short Text with One Action"
                               exampleUseTarget:self
                             exampleUseSelector:@selector(showSingleLineStyleBanner)];
   [bannerExampleList addObject:exampleUseInfo1];
 
   BannerExampleUseInfo *exampleUseInfo2 =
       [BannerExampleUseInfo infoWithIdentifier:@"example2"
-                                   displayName:@"MultiLine Aligned Button"
+                                   displayName:@"Short Text with One Action (Icon)"
                               exampleUseTarget:self
-                            exampleUseSelector:@selector(showMultiLineAlignedButtonStyleBanner)];
+                            exampleUseSelector:@selector(showSingleLineStyleBannerWithIcon)];
   [bannerExampleList addObject:exampleUseInfo2];
 
   BannerExampleUseInfo *exampleUseInfo3 =
       [BannerExampleUseInfo infoWithIdentifier:@"example3"
-                                   displayName:@"MultiLine Stacked Button"
+                                   displayName:@"Long Text with Two Short Actions"
```

Comment:
This isn't significantly longer text.  Consider doubling or trebling it.

Reply:
If it is doubled, the text will be trimmed on the screen since the limit of number of lines is 3.

Reply:
That sounds like a good example to show.

Reply:
Sure. I will add it. Tracked issue https://github.com/material-components/material-components-ios/issues/7017.